general/1_terminal/Ann-Horlacheva/homework

```diff
@@ -0,0 +1,46 @@
+Task 1
```

Comment:
This one is in `1_terminal/Ann-Horlacheva`.

Reply:
This one has the correct file location in a folder with my name.

Reply:
Then remove the other one.